components/esm/loadland.cpp

```diff
@@ -340,6 +342,10 @@ namespace ESM
         std::swap (mContext, land.mContext);
         std::swap (mDataTypes, land.mDataTypes);
         std::swap (mLandData, land.mLandData);
+        for (int i = 0; i < LAND_GLOBAL_MAP_LOD_SIZE; ++i)
+        {
+            std::swap (mWnam[i], land.mWnam[i]);
```

Comment:
Equally sized arrays can be swapped without a loop going through all of their elements.

Reply:
I thought it was only std::arrays, so I used this method during bug-hunt to keep track of values. Anyway, fixed in 2fdaacf . Also tested that it really does work.